### DISCUSSION TITLE
CLR20r3

### DISCUSSION THREAD

**Discussion by gdxdf2** [Meloong-Git/PCL#1689] [提问] [answered]
**"CLR20r3" appears when Windows 7 runs launcher**

**Comment by iPodtouch6557** [comment]
可以提交issue并且导出相关日志和截图，我们都不知道这是怎么出现的
You can submit an issue and export relevant logs and screenshots because we dont know how this happened

~还有我有俩朋友甚至把PCL2装到了手机上都能用~

**Reply by Chiloven945** [reply]
万一他是外国人

**Comment by WForst-Breeze** [comment, marked as answer]
请检查您的 .NET 运行环境后再次尝试运行 PCL2 启动器，您可以尝试修复或更新 .Net Framework 来解决此问题。
Please check your .NET running environment and try to re-run the PCL2.
